packages/atlas/src/views/viewer/PlaceBid/PlaceBid.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+import { ActionBar } from '@/components/ActionBar'
+import { Avatar } from '@/components/Avatar'
+import { Text } from '@/components/Text'
+import { JoyTokenIcon } from '@/components/_icons/JoyTokenIcon'
+import { cVar, media, sizes, zIndex } from '@/styles'
+
+const flexStyles = css`
+  display: flex;
+  align-items: center;
+`
+
+export const FlexWrapper = styled.div`
+  ${flexStyles};
+`
+
+export const Container = styled.div`
+  transform: translateY(100%);
+  opacity: 0;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  background-color: ${cVar('colorBackground')};
+  box-shadow: ${cVar('effectElevation24Layer1')}, ${cVar('effectElevation24Layer2')};
+  transition: transform ${cVar('animationTransitionSlow')}, opacity ${cVar('animationTransitionSlow')};
+  will-change: transform, opacity;
+  z-index: ${zIndex.globalOverlay};
+
+  &.place-bid {
+    &-exit,
+    &-enter-active,
+    &-enter-done {
+      opacity: 1;
+      transform: translateY(0);
+    }
+
+    &-enter,
+    &-exit-active {
+      transform: translateY(100%);
+      opacity: 0;
+    }
+  }
+`
+
+export const Content = styled.div`
+  padding-top: ${sizes(8)};
+  display: flex;
+  flex-direction: column-reverse;
+  height: 100%;
+  overflow-y: auto;
+  padding-bottom: 130px;
+
+  ${media.md} {
+    padding-top: ${sizes(16)};
+    flex-direction: row;
+    overflow-y: unset;
+    padding-bottom: unset;
+
+    > * {
+      flex: 50%;
+    }
+  }
+`
+
+export const NftPreview = styled.div`
+  display: flex;
+  justify-content: center;
+  flex: 1 0 auto;
+  align-items: baseline;
+  padding: 0 ${sizes(4)};
+
+  ${media.lg} {
+    flex: 50%;
+    padding: 0;
+  }
+`
+
+export const PlaceBidWrapper = styled.div`
+  padding-bottom: ${sizes(6)};
+
+  ${media.sm} {
+    height: calc(100% - 80px);
+    overflow-y: auto;
+    padding-bottom: unset;
+  }
+`
+
+export const InnerContainer = styled.div`
+  padding: 0 ${sizes(4)};
+
+  ${media.md} {
+    max-width: 550px;
+    padding-bottom: ${sizes(18)};
+  }
+  ${media.lg} {
+    max-width: 560px;
+  }
+`
+
+export const StyledJoyTokenIcon = styled(JoyTokenIcon)`
```

Comment:
I think we have a separate component with this filter already applied If I'm not mistaken.